kamelets/avro-deserialize-action.kamelet.yaml

```diff
@@ -45,7 +45,6 @@ spec:
         type: boolean
         default: true
         x-descriptors:
```

Comment:
Should we also remove x-descriptors when its empty

Reply:
Yes, it looks harder than what I thought